### PR TITLE
chore(make): add rules to build/run upstream-example

### PR DESCRIPTION
**Pull request by njhale**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 MOD_FLAGS := $(shell (go version | grep -q -E "1\.(11|12)") && echo -mod=vendor)
 CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} basename {}))
 
-.PHONY: build test vendor clean
+.PHONY: build test vendor clean run-example image-upstream clean-example
```
Comment (jpeeler) [inline Makefile]: I think you want example-image instead of image-upstream.

**Comment by njhale** [comment]
This has been sitting around long enough -- I'm also not sure if these rules will be useful in the future.

**Pull request closed without merging**